Textile/Textile.sublime-syntax

```diff
@@ -1,225 +1,1218 @@
+          ({{indentation_optional}})
+          ({{alignment_optional}})
+          ({{attributes_optional_class_style}})
+          (\.)
+          [ ](?=\S)
+        )
+      captures:
+        1: entity.name.tag.heading.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.heading.textile
+      push:
+        - meta_scope: markup.heading.6.textile
+        - meta_content_scope: meta.toc-list.textile
+        # before-eol-pop
+        - match: '$'
+          pop: 1
+        - include: inlines
+        - include: html
+
+###[ BLOCKS / PRE-FORMATTED TEXT ]#############################################
+
+  # https://textile-lang.com/doc/pre-formatted-text
+
+  pre-blocks:
+    - include: extended-blocks-pre
+    - include: blocks-pre
+
+  extended-blocks-pre:
+    - match: |-
+        (?x:
+          ^
+          (pre)
+          ({{indentation_optional}})
+          ({{alignment_optional}})
+          ({{attributes_optional_class_style}})
+          (\.\.)
+          [ ](?=\S)
+        )
+      captures:
+        1: entity.name.tag.pre.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.pre.textile
+      push: extended-block-pre-body
+
+  extended-block-pre-body:
+    - meta_scope: markup.raw.textile
+
+    - include: next-block-pop
+
+  blocks-pre:
+    - match: |-
+        (?x:
+          ^
+          (pre)
+          ({{indentation_optional}})
+          ({{alignment_optional}})
+          ({{attributes_optional_class_style}})
+          (\.)
+          [ ](?=\S)
+        )
+      captures:
+        1: entity.name.tag.pre.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.pre.textile
+      push: block-pre-body
+
+  block-pre-body:
+    - meta_scope: markup.raw.textile
+
+    - include: empty-line-pop
+
+###[ BLOCKS / NOTEXTILE ]######################################################
+
+  # https://textile-lang.com/doc/no-textile-processing
+
+  notextile-blocks:
+    - include: extended-blocks-notextile
+    - include: blocks-notextile
+
+  extended-blocks-notextile:
+    - match: |-
+        (?x:
+          ^
+          (notextile)
+          ({{indentation_optional}})
+          ({{alignment_optional}})
+          ({{attributes_optional_class_style}})
+          (\.\.)
+          [ ](?=\S)
+        )
+      captures:
+        1: entity.name.tag.notextile.textile
+        2: constant.other.indentation.textile
+        3: constant.other.alignment.textile
+        4: entity.name.type.textile
+        5: entity.name.tag.notextile.textile
+      push: extended-block-notextile-body
+
+  extended-block-notextile-body:
+    - meta_scope: text.plain
+
+    - include: next-block-pop
+    - include: html
```

Comment:
Same comments as for `<notextile>` tags apply here in general.

In particular scoping the block `text.plain` and supporting `html` highlighting is a conflict.

Is `html` highlighting also required within `<notextile><p>hello</p></notextile>` blocks?